content/workshops/openshift33/excerise-010-prerequisites.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
I see that the md pages are all a combination of exercise and title, change to just exercise number like this example;  exercisie1.0.md